Source/Scene/Cesium3DTileColorBlendMode.js

```diff
@@ -0,0 +1,41 @@
+/*global define*/
+define([
+        '../Core/freezeObject'
+    ], function(
+        freezeObject) {
+    'use strict';
+
+    /**
+     * Defines how per-feature colors set from the Cesium API or declarative styling blend with the source colors from
```

Comment:
These paragraphs need `<p></p>`; it currently looks like this:

![image](https://cloud.githubusercontent.com/assets/782098/26326588/c768ba02-3f09-11e7-9561-dab6771393d1.png)